app/src/main/java/com/example/garden_planner/fragments/GardenDetailFragment.java

```diff
@@ -106,6 +106,7 @@ public void onViewCreated(View view, Bundle savedInstanceState) {
             @Override
             public void onClick(View v) {
                 Intent i = new Intent(getContext(), EditGardenActivity.class);
```

Comment:
try not to use single character variable names. This can get very confusing when code gets more complex. Try to make the variable names describe the object or value they represent.

Reply:
Got it - I'll make sure to fix it when it autogenerates this code next time